Assets/Project/Modules/PlayerAnchor/Scripts/Player/View/ParticlesView/PlayerParticlesView.cs

```diff
@@ -52,21 +52,22 @@ public void PlayDashAnimation(float duration, Vector3 dashDirection)
 
         private async UniTaskVoid DoPlayDash(float duration, Vector3 dashDirection)
         {
+            Transform ghost = _particleFactory.Create(_config.DashGhostParticleType, Vector3.zero, quaternion.identity, _transformHolder);
             _particleFactory.Create(_config.DashDisappearParticleType, Vector3.zero, quaternion.identity, _transformHolder);
             await UniTask.Delay(TimeSpan.FromSeconds(Mathf.Max(0.0f, _config.TrailSpawnDelay)));
             
             Transform trail = _particleFactory.Create(_config.DashTrailParticleType, Vector3.zero, quaternion.identity, _transformHolder);
-            Vector3 trailScale = trail.localScale;
+            trail.localScale = Vector3.one;
             trail.DOLocalRotate(_config.DashTrailRotation, duration - _config.TrailSpawnDelay, RotateMode.LocalAxisAdd)
```

Comment:
Store this operation in a variable: `_config.DashTrailRotation, duration - _config.TrailSpawnDelay`
And, just to prevent errors, make a Mathf.Max() to avoid negative numbers